code/datums/components/sitcomlaughter.dm

```diff
@@ -0,0 +1,22 @@
+/datum/component/wearertargeting/sitcomlaughter
+	valid_slots = list(ITEM_SLOT_HANDS, ITEM_SLOT_BELT, ITEM_SLOT_ID, ITEM_SLOT_LPOCKET, ITEM_SLOT_RPOCKET, ITEM_SLOT_SUITSTORE, ITEM_SLOT_DEX_STORAGE)
+	signals = list(COMSIG_MOB_CREAMED, COMSIG_ON_CARBON_SLIP, COMSIG_ON_VENDOR_CRUSH, COMSIG_MOB_CLUMSY_SHOOT_FOOT)
```

Comment:
```suggestion
	signals = list(COMSIG_MOB_CREAMED, COMSIG_ON_CARBON_SLIP, COMSIG_ON_VENDOR_CRUSH, COMSIG_MOB_CLUMSY_SHOOT_FOOT, COMSIG_MOB_DEATH)
```
Some other possibilities, while we're here:
- Missing a tackle (line 374, tackle.dm)
- Impaling yourself on a desword (line 131, dualsaber.dm)
- Handcuffing yourself (line 48, handcuffs.dm)
basically anywhere there's a TRAIT_CLUMSY check. You could probably make a generic signal for "something funny happened" instead of making a ton of different signals for each snowflake clown interaction

Reply:
the point of signals theres a crapton of them, one for each event that occurs (e.g. getting creamed, shooting yourself in the foot), that specifically makes them non-snowflake.,

Reply:
seems a little weird to me but alright
rest of the suggestion stands